help.go

```diff
@@ -476,3 +482,14 @@ func wrapLine(input string, offset int, wrapAt int, padding string) string {
 func offset(input string, fixed int) int {
 	return len(input) + fixed
 }
+
```

Comment:
I think this function deserves a comment.

Reply:
Yes.